cfme/tests/services/test_service_customer_bz.py

```diff
@@ -115,3 +116,48 @@ def test_refresh_dynamic_field(appliance, import_datastore, import_data,
     view.fields(ele_label).fill("Password")
     # If Refresh works submit button will enable
     wait_for(lambda: not view.submit_button.disabled, timeout=7)
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[BZ(1570152)])
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize(
+    "import_data",
+    [DatastoreImport("bz_1570152.zip", "bz_1570152", None)],
+    ids=["sample_domain"],
+)
+# Parametrizing for import_dialog fixture
+@pytest.mark.parametrize("file_name", ["bz_1570152.yml"],
+    ids=["sample_dialog"],
+)
+def test_update_dynamic_checkbox_field(appliance, import_datastore, import_data,
+                               catalog_item_with_imported_dialog):
+    """Tests update dynamic check box field when Selecting true in dropdown.
+    Metadata:
+        test_flag: provision
+
+    Bugzilla:
+        1570152
+
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        initialEstimate: 1/8h
+        tags: service
+    setup:
+        1. Import bz dialog
+        2. Import  bz datastore
+        3. Create catalog item
+    testSteps:
+        1. Order service catalog and Select True from dropdown
+    expectedResults:
+        1. Selecting true from dropdown field should update dynamic check box field
+    """
+    cat_item, _ = catalog_item_with_imported_dialog
```

Comment:
The second item of catalog_item_with_imported_dialog returns the element_label of dialog  which can be used  in line 161 instead of "trigger "
so you can do 

        cat_item, ele_label = catalog_item_with_imported_dialog
        view.fields(ele_label).dropdown.fill("true")